langchain/chains/conversation.ts

```diff
@@ -1,10 +1,10 @@
 import { LLMChain } from "./index";
 import { BaseLLM } from "../llms";
-import { BasePromptTemplate , PromptTemplate } from "../prompt";
+import { BasePromptTemplate, PromptTemplate } from "../prompt";
 
 import { BaseMemory, BufferMemory } from "../memory";
 
-
+// eslint-disable-next-line max-len
```

Comment:
We might want to remove this eslint rule? Max length is what prettier is here for. Thoughts? @sullivan-sean

Reply:
Sure, I'm not picky about linting let's leave this to prettier!